src/cmd/cli/command/commands.go

```diff
@@ -56,6 +56,27 @@ func prettyError(err error) error {
 	return err
 }
 
+func interactiveComposeDown(ctx context.Context, c cliClient.Client, projectName string) error {
+	var wantComposeDown bool
+	err := survey.AskOne(&survey.Confirm{
+		Message: "Run 'compose down' to deactivate project: " + projectName + "?",
+	}, &wantComposeDown)
+
+	if err != nil {
+		return err
```

Comment:
nobody checks this `err`, so either check in the caller (and `term.Debug`?) or print something here.